tests/accessor/local_accessor_access_among_work_items.h

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for the properties of sycl::local_accessor that can access to
```

Comment:
Do we really need word "properties" here?

Reply:
No, deleted in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/364/commits/5db87bf32e0364c53ccc4bcf6341571d4c8c3a19